lib/community.js

```diff
@@ -451,6 +451,13 @@ exports.all = {
         quote: 'I thought hapi was impressive for its community support, reliability, and rate of adoption by other organizations and developers. As a scrappy, low-budget, non-profit charity, we want to leverage our volunteer base, which includes developers who can plug in and help out with building and modifying our web applications. hapi seems like the future that a lot of Node coders are going to know.',
         person: 'Steev Hise, Senior Engineer'
     },
+    'Theodo UK': {
+        url: 'http://theodo.co.uk',
+        logo: 'logo-theodo.jpg',
+        width: 150,
+        quote: 'I came from a background in Loopback and wanted to try another approach. We went for Hapi as the backend for a mobile project and weren\'t disappointed. Will definitel use it on other projects.',
```

Comment:
Thanks for the kind words!  Just two minor corrections,

1. Found a little typo, `definitel` --> `definitely`
2. `Hapi` --> `hapi`.  By convention we always stylize hapi to be all lowercase :)